src/vector_of_arrays.jl

```diff
@@ -490,7 +490,7 @@ function consgrouped_ptrs end
 export consgrouped_ptrs
 
 function consgrouped_ptrs(A::AbstractVector)
-    elem_ptr = Vector{Int}()
+    elem_ptr = sizehint!(Int32[], length(A))
```

Comment:
We should keep the `Vector{Int}()` here - using `sizehint!` can be very wasteful here, since the length of `elem_ptr` can be significantly smaller than the length of `A`. And `Int` is the Julia default for indices.